.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Crates
+
+on:
+  release:
+    types: [ published ]
+
+env:
+  CARGO_TERM_COLOR: always
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  RUSTFLAGS: "-D warnings"
+  RUSTUP_MAX_RETRIES: 10
+  RUST_BACKTRACE: 1
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Not using `Swatinem/rust-cache@v2`?

Reply:
Thanks, I copy-pasted the initial workflow from `bench.yml` and forgot to add it